sources/packages/api/src/route-controllers/application/models/application.model.ts

```diff
@@ -39,6 +72,31 @@ export interface ApplicationDataDto {
    * Program Year is added to integrate the application in camunda workflows
    */
   programYear: string;
+
+  /**
+   * Offering details fro student application
```

Comment:
Please fix the typo "fro" on this and other comments.
Add a "." period to the sentence.